public/components/Glossary/Glossary.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const MedianAttentionTimeExplainerText = 'Attention time is the value where a user is knowingly active on the page; they have recently moved their mouse or scrolled.';
```

Comment:
Perhaps `... is the measure of how long a user ...` ?

Reply:
Is it worth saying how long recent is, or is that deliberately left unstated?

Reply:
Yes, that sounds better...
The time is left deliberately unstated as I don't know the exact time you need to be inactive for